WordPress/Classes/ViewRelated/Views/List/NotificationsList/NotificationsTableViewCellContent.swift

```diff
@@ -0,0 +1,122 @@
+import SwiftUI
+import DesignSystem
+
+extension NotificationsTableViewCell {
+    struct Content: View {
+        private let title: String
```

Comment:
[Question] In the existing implementation, the notification title is the output of this method `notification.renderSubject()`, which returns an `NSAttributedString`, but the `Content.title` is a `String`. How's that going to work exactly?

![CleanShot 2024-02-08 at 18 21 42](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/56e52dcc-f719-4fd4-8b17-8e00f19afc81)

Reply:
This won't be necessary due to the fact that the title no longer contains and font diversity. All of it is in regular font weight.